homework-submission/week4hw/src/stores/TodoListStore.js

```diff
@@ -0,0 +1,65 @@
+import { observable, action } from "mobx";
+import todosList from "./../data/todos.json";
+
+class TodosListStore {
+  @observable listItems = todosList;
+  @observable isEditing = false;
+
+  @action
+  onCheckboxChange = index => {
+    console.log("hhhh");
+    console.log(this.listItems);
+    if (index >= 0) {
+      this.listItems[index].done = !this.listItems[index].done;
+    }
+    console.log(this.listItems);
+  };
+
+  @action
+  onSubmitForm = event => {
+    event.preventDefault();
+
+    let id = 0;
+    if (this.listItems.length > 0) {
+      id = this.listItems[this.listItems.length - 1].id + 1;
+    }
+
+    let deadline = event.target.deadline.value;
+    let description = event.target.description.value;
```

Comment:
if you will not change the value put it as `const` otherwise use `let`.

Reply:
Yep, they should be `const`